shared-libs/transitions/src/transitions/death_reporting.js

```diff
@@ -65,29 +48,20 @@ module.exports = {
       doc.form &&
       doc.type === 'data_record' &&
       (isConfirmForm(doc.form) || isUndoForm(doc.form)) &&
-      doc.fields &&
-      doc.fields.patient_id &&
+      doc.patient &&
```

Comment:
interesting ... i didn't realize every doc that went through sentinel transitions was hydrated before being passed to filter. good ot know